src/stravalib/tests/unit/test_validate_activity_type.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+
+@pytest.fixture
+def params():
+    """A fixture that creates a dictionary representing
+    parameters associated with a Strava activity."""
+
+    params = {
+        "name": "New Activity",
+        "start_date_local": "2024-03-04T18:56:47Z",
+        "elapsed_time": 9000,
+        "description": "An activity description here.",
+        "distance": 5700,
+    }
+    return params
+
+
+@pytest.mark.parametrize(
+    "sport_type, activity_type, expected_result, expected_exception, create",
+    (
+        ("TrailRun", None, "TrailRun", None, False),
```

Comment:
As you mentioned somewhere, let's add a case here where neither `activity_type` nor `sport_type` are given, and we expect a `ValueError`.